<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventQtSlotConnect</class>
 <widget class="QMainWindow" name="EventQtSlotConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EventQtSlotConnect</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>551</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Cmrep Skeleton</string>
     </attribute>
     <widget class="QPushButton" name="cmrepVskel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>440</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Skeleton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>e</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>p</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>c</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>t</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="eParameter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="pParameter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="cParameter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="tParameter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>60</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pruning Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>240</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>s</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>q</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>General Option</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pathParameter">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Pick</string>
     </attribute>
     <widget class="QGroupBox" name="TaggroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Tag</string>
      </property>
      <widget class="QComboBox" name="comboBoxTagPoint">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="maxVisibleItems">
        <number>10</number>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAddTag">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Tag</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDeleteTag">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Tag</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonEditTag">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit Tag</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelPoint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Point Number:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="PointNumber">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelTriangle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Triangle Number:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="TriangleNumber">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="SkandMeshgroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Skeleton and Mesh</string>
      </property>
      <widget class="QCheckBox" name="checkBoxHideMesh">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hide Mesh</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxHideSkel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hide Skeleton</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>231</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Decimation</string>
      </property>
      <widget class="QSlider" name="TargetReductSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="TargetReductLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Target Reduction</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Feature Angle</string>
       </property>
      </widget>
      <widget class="QSlider" name="FeatureAngleSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="FeatureAngleLineEdit">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ApplyDecimateButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply Decimation</string>
       </property>
      </widget>
     </widget>
     <zorder>SkandMeshgroupBox</zorder>
     <zorder>TaggroupBox</zorder>
     <zorder>labelPoint</zorder>
     <zorder>PointNumber</zorder>
     <zorder>labelTriangle</zorder>
     <zorder>TriangleNumber</zorder>
     <zorder>groupBox</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Save Option</string>
     </attribute>
     <widget class="QGroupBox" name="SaveOptiongroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Save Option</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>Grid Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="GridTypeComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Subdivision Level</string>
       </property>
      </widget>
      <widget class="QComboBox" name="SubLevelComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>41</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>43</x>
         <y>110</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Solver Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="SolverTypeComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Constant Rho</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>90</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="RhoLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>-0.001</string>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="ConsRadiusCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Constant Radius</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="RadiusLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="cmrep_progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>qvtkWidget</zorder>
   <zorder>cmrep_progressBar</zorder>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
